frontend/src/components/pages/CampsPage.tsx

```diff
@@ -1,9 +1,85 @@
 import React, { useContext } from "react";
+import WaitlistedCampersTable from "./CampOverview/WaitlistedCampersTable";
+
+const waitlistedCampers = [
+  {
```

Comment:
Similar to my comment from before, instead of making these as regular JSON objects, I think it's better practise to define a type and use a list of objects instead. I think this list of WaitlistCamperType objects would be constructed when the get request from the Camp Overview page returns the data